data/places/tolPlacesWithTypes.ttl

```diff
@@ -3384,52 +3384,6 @@ told:_Place_1753
 	gist:isCategorizedBy tolx:_PlaceType_cave ;
 	.
 
-told:_Place_1755
```

Comment:
@treasuryoflives Just checking to make sure you wanted to delete all of these places.